firehose.c

```diff
@@ -153,7 +153,8 @@ static int firehose_read(struct qdl_device *qdl, int wait, int (*response_parser
 			for (node = nodes; node; node = node->next) {
 				if (xmlStrcmp(node->name, (xmlChar*)"log") == 0) {
 					firehose_response_log(node);
-				} else if (xmlStrcmp(node->name, (xmlChar*)"response") == 0) {
+				}
+				else if (xmlStrcmp(node->name, (xmlChar*)"response") == 0) {
```

Comment:
please do not change code style. or if you do , then you need to do it consistently and in a separate patch anyways.

Reply:
I have updated patch with reverted code style.